x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts

```diff
@@ -20,7 +20,7 @@ export default createTestConfig({
   suiteTags: { exclude: ['skipSvlOblt'] },
   services,
   // add feature flags
-  kbnServerArgs: ['--xpack.infra.enabled=true'],
+  kbnServerArgs: ['--xpack.infra.enabled=true', '--xpack.security.roleManagementEnabled=true'],
```

Comment:
__question:__ is your plan to also set this flag for the remaining project types tests (security and search)?

Reply:
Chose 1 project type for each feature flag suite, I thought it was redundant to add them to all projects, but these are pretty fast tests, I will add them to all project types

Reply:
Fixed in latest commit